src/main/resources/db/migration/V9__consumed_event_marker.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE supplier_event
+    ADD COLUMN event_consumed BOOLEAN NOT NULL DEFAULT FALSE;
```

Comment:
Do we want to add migration logic to mark all the ones that should be consumed as consumed?

Reply:
yes good point